src/lib/client.ts

```diff
@@ -50,7 +50,7 @@ export class Client {
         const reward = this.api.registry.createType("u128", data[1]);
         blockReward = Number((reward.toBigInt() * 100n) / SUNIT) / 100;
       } else if (section === "transactionFees") {
-        // TODO
+        // TODO: include storage and compute fees
```

Comment:
And voting :slightly_smiling_face:

Reply:
Voting is already in progress in another branch